Package.swift

```diff
@@ -121,7 +121,14 @@ var targets: [Target] = [
             .product(name: "SwiftSyntax", package: "swift-syntax"),
             .product(name: "SwiftCompilerPlugin", package: "swift-syntax"),
         ],
-        swiftSettings: [.swiftLanguageMode(.v5)]
+        swiftSettings: [
+            .swiftLanguageMode(.v5),
+            .unsafeFlags(
+                [
```

Comment:
Without this, the compiler was complaining about there being a main() entrypoint in this target which clashes